Update rec. to not use `any` paired with `maskable` (#3892)
GoogleChrome/web.dev: src/site/content/en/blog/maskable-icon/index.md

## Are my current icons ready?

Since maskable icons need to support a variety of shapes, you supply an opaque image with some
padding that the browser can later crop into the desired shape and size. It's best not to rely on

> any particular shape, since it can vary by browser and per platform.

Author: ```suggestion
any particular shape, since the ultimately chosen shape can vary by browser and per platform.
```

> any particular shape, since it can vary by browser and per platform.

Author: The shape. I have updated the sentence.

  </figcaption>
</figure>

> If you want to preview your maskable icon in other shapes it may appear in on Android, I've created
> a tool called [Maskable.app](https://maskable.app/). Open an icon, then Maskable.app will let you

Reviewer: ```suggestion
To test your maskable icon with the variety of Android shapes, use the
[Maskable.app](https://maskable.app/) tool I've created.
Open an icon, then Maskable.app will let you
```

Reviewer: I had a little trouble following the sentence.

try out various shapes and sizes, and you can share the preview with others on your team.

## How do I adopt maskable icons?

> If you want to create a maskable icon based off your existing icon, you can use the

Reviewer: ```suggestion
If you want to create a maskable icon based on your existing icon, you can use the
```

Reviewer: More idiomatic.

Once you've created a maskable icon image and tested it out in DevTools, you'll need to update your
[Web App Manifest](https://developers.google.com/web/fundamentals/web-app-manifest) to point to the
new assets. The Web App Manifest provides information about your web app in a JSON file, and
includes an [`icons` array](https://developers.google.com/web/fundamentals/web-app-manifest#icons).

With the inclusion of maskable icons, a new property value has been added for image resources listed
in a Web App Manifest. The `purpose` field tells the browser how your icon should be used. By
default, icons will have a purpose of `"any"`. These icons will be resized on top of a white
background on Android.

Maskable icons should use a different purpose: `"maskable"`. This indicates that an image is meant
to be used with icon masks, giving you more control over the result. This way, your icons will not
have a white background. You can also specify multiple space-separated purposes (for example,
`"any maskable"`), if you want your maskable icon to be used without a mask on other devices.

{% Aside %}
  While you _can_ specify multiple space-separated purposes like `"any maskable"`, in

>   practice you _shouldn't_, since using `"maskable"` icons as `"any"` icons is suboptimal as the icon

Reviewer: ```suggestion
  practice you _shouldn't_. Using `"maskable"` icons as `"any"` icons is suboptimal as the icon
```

Reviewer: Give the reader a chance to 'breath' before moving to the explanation. Also having the recommendation stand alone gives it more force. I didn't start the next sentence with "This is because" because we expect the next sentence to be the reason and it fulfills that expectation.

  is going to be used as-is, resulting in excess padding and making the core icon content smaller.

>   Ideally, icons for the `"any"` purpose should have the extra padding from the maskable icon removed

Reviewer 2: ```suggestion
  Ideally, icons for the `"any"` purpose should transparent regions and no extra padding, like your site's favicons. 
```
Might be worthwhile to compare "any" to favicons, as most sites follow that system. Also worthwhile to note that "any" icons don't have to be square/opaque.

new assets. The Web App Manifest provides information about your web app in a JSON file, and
includes an [`icons` array](https://developers.google.com/web/fundamentals/web-app-manifest#icons).

With the inclusion of maskable icons, a new property value has been added for image resources listed
in a Web App Manifest. The `purpose` field tells the browser how your icon should be used. By
default, icons will have a purpose of `"any"`. These icons will be resized on top of a white
background on Android.

Maskable icons should use a different purpose: `"maskable"`. This indicates that an image is meant
to be used with icon masks, giving you more control over the result. This way, your icons will not
have a white background. You can also specify multiple space-separated purposes (for example,
`"any maskable"`), if you want your maskable icon to be used without a mask on other devices.

{% Aside %}
  While you _can_ specify multiple space-separated purposes like `"any maskable"`, in
  practice you _shouldn't_. Using `"maskable"` icons as `"any"` icons is suboptimal as the icon
  is going to be used as-is, resulting in excess padding and making the core icon content smaller.

>   Ideally, icons for the `"any"` purpose should have transparent regions and no extra padding, like your site's favicons. 
>   since the browser isn't going to do that for them.

Reviewer: ```suggestion
  Ideally, icons for the `"any"` purpose should have transparent regions and no extra padding, like your site's favicons,
   since the browser isn't going to do that for them.
```
